swift_browser_ui_frontend/src/views/Upload.vue

```diff
@@ -2,215 +2,296 @@
   <div
     id="uploadform"
     class="contents"
-  >
+  > 
+    <b-message
+      v-if="$te('message.keys') && fixedrecvKeys.length > 0"
+      type="is-info"
+    >
+      {{ $t('message.keys.defaultKeysMessage') }}
```

Comment:
I'd much prefer not having the defaultKeysMessage under `message.keys`, but preferably `message.encrypt` as having it in this location confuses the key fetch logic – this approach will lead to a single undefined address fetch in the current implementation I think. I should probably stop using the `lang.js` file for general config usage since that's probably confusing, and create a new file for that purpose in the future.